pkg/controllers/managementuser/resourcequota/resource_quota_sync.go

```diff
@@ -457,34 +417,26 @@ func (c *SyncController) getResourceLimitToUpdate(ns *corev1.Namespace) (*corev1
 	}
 }
 
-func completeQuota(existingQuota *v32.NamespaceResourceQuota, defaultQuota *v32.NamespaceResourceQuota) (*v32.NamespaceResourceQuota, error) {
-	if defaultQuota == nil {
+func completeQuota(requestedQuota *v32.ResourceQuotaLimit, defaultQuota *v32.ResourceQuotaLimit) (*v32.ResourceQuotaLimit, error) {
```

Comment:
Discussed with @cbron - this is normal behavior. Potentially this can be counter-intuitive for some users.
In the UI, however, users can request fewer configMaps than the default, which they can see.

Reply:
IMO it makes a lot of sense to make the default option a safe option (i.e. - if we need you to set a value it should be required to set the value, otherwise, it should always be safe to use the default). 

But that's just my opinion - as long as we are aware of (and ok with) the solution not much more to say. Feel free to resolve.